js/ui/models/site.es6.js

```diff
@@ -45,6 +46,7 @@ Site.prototype = $.extend({},
                           }
                           this.setSiteProperties()
                           this.setHttpsMessage()
+                          this.tosdr = backgroundTabObj.site.score.tosdr
```

Comment:
This needs to go above here, inside of the guard `if (backgroundTabObj) ...`

Reply:
Yes, thanks.